.gitignore

```diff
@@ -17,5 +17,4 @@ distribute-*.tar.gz
 up.bat
 anthrocon_plugin/
 add_table_extras.sql
-alembic*
```

Comment:
Minor note: because of how these PRs were done, stuff like the alembic files ended up in tons of different PRs.  That makes it hard for people looking through them (especially weeks after the fact as I'm trying to get caught up on any suggestions/comments for future reference) to see what actually changed as part of the PR.  I suggest doing PR chains to handle this kind of thing in general, e.g. PR from Feature B into Feature A and then PR Feature A into your working branch.